src/modules/ProxiedModuleApi.ts

```diff
@@ -81,23 +81,22 @@ export class ProxiedModuleApi implements ModuleApi {
     /**
      * @override
      */
-    public openDialog<
-        M extends object,
-        P extends DialogProps = DialogProps,
-        C extends React.Component = React.Component,
-    >(
+    public openDialog<M extends object, P extends DialogProps, C extends DialogContent<P>>(
```

Comment:
TS Looks unhappy here

Reply:
the TSC strict CI is mistaken, the normal `tsc` pass is happy here